README.rst

```diff
@@ -203,72 +203,3 @@ to build a toc without it rendering in your index.
 By default, the navigation will "stick" to the screen as you scroll. However if your toc
 is vertically too large, it will revert to static positioning. To disable the sticky nav
 altogether change the setting in ``conf.py``.
-
-Contributing or modifying the theme
-===================================
-
-The sphinx_rtd_theme is primarily a sass_ project that requires a few other sass libraries. I'm
-using bower_ to manage these dependencies and sass_ to build the css. The good news is
-I have a very nice set of grunt_ operations that will not only load these dependencies, but watch
-for changes, rebuild the sphinx demo docs and build a distributable version of the theme.
-The bad news is this means you'll need to set up your environment similar to that
-of a front-end developer (vs. that of a python developer). That means installing node and ruby.
-
-Set up your environment
------------------------
-
-1. Install sphinx_ into a virtual environment.
-
-.. code::
-
-    pip install sphinx
-
-2. Install sass
-
-.. code::
-
-    gem install sass
-
-2. Install node, bower and grunt.
-
-.. code::
-
-    // Install node
-    brew install node
-
-    // Install bower and grunt
-    npm install -g bower grunt-cli
-
-    // Now that everything is installed, let's install the theme dependecies.
-    npm install
-
-Now that our environment is set up, make sure you're in your virtual environment, go to
-this repository in your terminal and run grunt:
-
-.. code::
-
-    grunt
-
-This default task will do the following **very cool things that make it worth the trouble**.
-
-1. It'll install and update any bower dependencies.
-2. It'll run sphinx and build new docs.
-3. It'll watch for changes to the sass files and build css from the changes.
-4. It'll rebuild the sphinx docs anytime it notices a change to .rst, .html, .js
-   or .css files.
-
-Before you create an issue
---------------------------
-
-I don't have a lot of time to maintain this project due to other responsibilities.
-I know there are a lot of Python engineers out there that can't code sass / css and
-are unable to submit pull requests. That said, submitting random style bugs without
-at least providing sample documentation that replicates your problem is a good
-way for me to ignore your request. RST unfortunately can spit out a lot of things
-in a lot of ways. I don't have time to research your problem for you, but I do
-have time to fix the actual styling issue if you can replicate the problem for me.
-
-TODO
-====
-
-* Separate some sass variables at the theme level so you can overwrite some basic colors.
```

Comment:
We shouldn't be removing content from our docs. Using an reST include here, we can include this content from another source, such as a `CONTRIBUTING.rst`